content/en/docs/reference/config/security/authorization-policy/index.html

```diff
@@ -195,7 +195,7 @@ <h2 id="Condition">Condition</h2>
 <td><code>string</code></td>
 <td>
 <p>The name of an Istio attribute.
-See the <a href="/docs/reference/config/">full list of supported attributes</a>.</p>
+See the <a href="/docs/reference/config/security/conditions">full list of supported attributes</a>.</p>
```

Comment:
This is generated doc. It needs to be fixed in the .proto file in the istio/api repository.